angular-ngrx-scss/src/app/state/user/user.effects.spec.ts

```diff
@@ -2,18 +2,19 @@ import { TestBed } from '@angular/core/testing';
 import { provideMockActions } from '@ngrx/effects/testing';
 import { Action } from '@ngrx/store';
 import { Observable, of } from 'rxjs';
+import { fetchUserData, fetchUserDataSuccess } from '.';
```

Comment:
This should still have `/user.actions`, to keep things clear where this comes from

Reply:
Whoops. That's what happens when I don't check IDE auto-imports, haha.